starter-code/main.js

```diff
@@ -0,0 +1,129 @@
+const MongoDB = require('mongodb');
+const mongoClient = MongoDB.MongoClient;
+const clear = require('clear');
+const readline = require('readline');
+const rl = readline.createInterface({
+	input: process.stdin,
+	output: process.stdout
+});
+
+const url = `mongodb://localhost:27017/crunchbase`
+
+mongoClient.connect(url, (error, db) => {
+    if (error) {
+      console.log('Error trying to connect to the Database');
+      console.log(error);
+    } else {
+      console.log('Connection established correctly!! 😬');
+  
+      function mainMenu(){
+        clear();
+        printMenu();
+        rl.question('Type an option: ', (option) => {
+          switch(option){
+            case "1":
+              db.collection('companies').find({}, {name: 1, _id: 0}).toArray((error, result) => {
+                if (error) {
+                  console.log(error);
+                  rl.question(`\nType ERROR enter to continue: `, (answer) => { mainMenu() });
+                } else {
+                  console.log(result); 
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                }
+              });
+              break;
+            case "2":
+                let count = db.collection('companies').count();
+                console.log (count);
+                rl.question(`\nType enter to continue: `, (count) => {mainMenu()});
+                break;
+            case "3":
+                let founded = db.collection('companies').count({"founded_year":{$eq: 2004}});
+                console.log(founded);
+                rl.question(`\nType enter to continue: `, (founded) => {mainMenu()});
+                break;
+            case "4":
+                db.collection('companies').find({$and: [{"founded_year":{$eq: 2004}},{"founded_month":{$eq: 2}}]}, {name: 1,founded_year:1,founded_month:1, _id: 0}).toArray((error, result) => {
```

Comment:
You don't need the $and here, when you put more than one query (for different criteria) separated by commas, a logic and is applied.